komodo/post_messages.py

```diff
@@ -0,0 +1,154 @@
+from argparse import ArgumentParser
+import yaml as yml
+import fnmatch
+import sys
+import os
+import shutil
+import hashlib
+
+
+def get_messages_and_scripts(release_name, motd_db):
+    scripts = []
+    messages = []
+    inline = []
+    for key in motd_db:
+        if fnmatch.fnmatch(release_name, key):
+            scripts = (
+                scripts + motd_db[key].get("scripts")
+                if motd_db[key].get("scripts") is not None
+                else scripts
+            )
+            messages = (
+                messages + motd_db[key].get("messages")
+                if motd_db[key].get("messages") is not None
+                else messages
+            )
+            inline = (
+                inline + motd_db[key].get("inline")
+                if motd_db[key].get("inline") is not None
+                else inline
+            )
+
+    return scripts, messages, inline
+
+
+def create_inline_messages(messages, clear_old_messages, dst_path):
+    if clear_old_messages:
+        if os.path.isdir(dst_path):
+            shutil.rmtree(dst_path)
+        os.makedirs(dst_path)
+
+    for msg in messages:
+        filename = hashlib.md5(msg.encode()).hexdigest()
+        filename = "0Z" + filename  # for orderings sake
+        with open(os.path.join(dst_path, filename), "w") as new_file:
+            new_file.write(msg)
+
+
+def copy_files(file_list, dst_path, src_path):
+    if os.path.isdir(dst_path):
+        shutil.rmtree(dst_path)
+
+    os.makedirs(dst_path)
+    for file_name in file_list:
+        print("Installing message: {}".format(file_name))
+        file_path = os.path.join(src_path, file_name)
+        if not os.path.exists(file_path):
+            raise SystemExit("ERROR: Message file {} does not exisit".format(file_name))
+        shutil.copy(file_path, os.path.join(dst_path, file_name))
+
+
+def get_parser():
+    parser = ArgumentParser(description="Post messages to a release.")
+
+    parser.add_argument(
+        "--releases", "-r", nargs="+", help="The releases to post messages to.",
+    )
+
+    parser.add_argument(
+        "--motd-db", "-m", required=True, help="YML file defining the messages.",
+    )
+
+    parser.add_argument(
+        "--komodo-prefix",
+        "-k",
+        required=True,
+        help="Path to folder holding komodo-releases.",
```

Comment:
Ambiguous: Path to `https://github.com/equinor/komodo-releases` or `/prog/res/komodo`? Perhaps `"Path to komodo releases root (eg. /prog/res/komodo)"`. Also make it the default.